Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml

```diff
@@ -3,6 +3,11 @@ commonfields:
   id: OpenPhish_v2
   version: -1
 configuration:
+- defaultvalue: true
```

Comment:
```suggestion
- defaultvalue: false
```

Reply:
@Itay4 integration configuration does not change on an integration update, or am I missing anything?

Reply:
it does

Reply:
we talked, decided to make the default as false.